.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,15 @@
+name: Release Drafter
```

Comment:
What does this do ?

Reply:
Creates a draft release if a pull request is merged.

Reply:
We don't need this, as a pull request being merged is not the trigger for a release, and it could create a bunch a unneeded releases.